Main.java

```diff
@@ -9,7 +10,36 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+   
+   Scanner input = new Scanner (System.in);
+    
+    // how many chromebook chargers 
+    System.out.println("How many chromebook chargers?");
+    
+    //ammount of chargers 
+      int A = input.nextInt();
+
+    // ammount of motherboards 
+    System.out.println("How many motherboards?");
+
+    //ammount of motherboards 
+    int B = input.nextInt();
+    
+    //amount mice
+    System.out.println("How many mice?");
+    
+    //number of mice
+    int C = input.nextInt();
```

Comment:
Remember, variable names should always begin with a lowercase letter and describe what they are storing. A should be called numChromebookChargers or something along those lines. B and C are similar.